### ISSUE TITLE
Missing PDOM items in dev version 1.4.0-dev.1

### ISSUE THREAD

**Issue by terracoda** [phetsims/balloons-and-static-electricity#290]
Just happened to notice that a few things are missing from the PDOM in version 1.4.0-dev.1, so just checking to make sure they are in the next release which I think you are working on, @jessegreenberg.

Using VO with Safari on MacOS X 10.12.3

- [x] Wall button missing on-demand help text.
- [x] Wall button's label content seems to be wrapped in a p-tag. The p-tag is not necessary.
- [x] Two-balloon experiment toggle button is missing it's label content. I'm not sure if this is supposed to be a button or a checkbox with role switch?
- [x] Two-balloon switch/toggle also missing on-demand help text.

Also:
- [x] Balloon Settings h3 missing (but not sure if this was a design change, or possibly never implemented due to simplified sim used in early interviews).

I think the mark-up should be something like this:
For Wall button
```html
<button>Remove Wall</button>
<p>Play with or without the wall.</p>
```
And for Two-balloon Experiments
```html
<h3>Balloon Settings</h3>
<button aria-pressed="false">Two-balloon Experiments</button>
<p>Play with two balloons or just one.</p>
```
OR
```html
<h3>Balloon Settings</h3>
<label for="balloon-on-off">Two-balloon Experiments</label>
<input id="balloon-on-off" type=checkbox role="switch" >
<p>Play with two balloons or just one.</p>
``` 
**Note** the Rest Balloon button and help text look fine. 

**Alternatives that might simplify Control Panel structure:**
We could discuss this in a separate issue. The goal is just to make the structure and reading of the control panel items more parallel.
- Use `aria-label` on Charge Settings UL, instead of an `h3`, 
- group the two items under the Balloon Settings heading in a `ul` list, and also label them with an `aria-label`.

**Comment by jessegreenberg** [comment]
Thanks @terracoda, I will work on this before the next version.

**Comment by terracoda** [comment]
@jessegreenberg, and just to be clear the potential alternatives for the control panel are low priority. I think you and I should discuss that first.

**Comment by jessegreenberg** [comment]
> Wall button's label content seems to be wrapped in a p-tag. The p-tag is not necessary.

It isn't necessary, but @terracoda is it causing problems?  This is happening because of the implementation in scenery-phet/MultiLineText.

**Comment by jessegreenberg** [comment]
We can override the options passed to scenery-phet/MultiLineText, but I am more just curious if a ```<p>``` tag inside of a button tag is generally problematic.

**Comment by jessegreenberg** [comment]
I added the balloon settings content to the PDOM:
![balloon settings](https://user-images.githubusercontent.com/6396244/30228880-e3016604-949c-11e7-9e65-99aa978c2b53.PNG)

**Comment by jessegreenberg** [comment]
Regarding https://github.com/phetsims/balloons-and-static-electricity/issues/290#issuecomment-328192099, we would need to set label and tagName to null for the MultiLineTexts, and add a function that update the accessible label on the wall button. At the moment, this is all being handled generally by sun/ToggleNode.  So it would be good to hear @terracoda response to https://github.com/phetsims/balloons-and-static-electricity/issues/290#issuecomment-328185141https://github.com/phetsims/balloons-and-static-electricity/issues/290#issuecomment-328185141 before changing.

**Comment by terracoda** [comment]
@jessegreenberg, regarding the https://github.com/phetsims/balloons-and-static-electricity/issues/290#issuecomment-328192099, it is likely not a problem. It's not a good practice to put block-level elements inside inline elements. 

Can you validate the code in the pdom to see if validates with errors?

**Comment by terracoda** [comment]
@jessegreenberg, wrapping a p-tag inside a button is indeed invalid `html` code. Errors in markup code can cause inconsistent rendering, and possible confusion in the accessibility tree. It would good to fix the general handling of making a toggle button.

![button-p-error](https://user-images.githubusercontent.com/1902116/30244296-116ee9f4-9596-11e7-8f11-f8baf0f31991.png)

**Comment by terracoda** [comment]
If the `p` element is needed, it would have to go around the `button`, rather than within it.

**Comment by terracoda** [comment]
[Article from MDN on flow and phrasing content](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Content_categories#Flow_content) (new content categories for HTML5).

Generally speaking, what were block-level elements  now are referred to as containing _flow content_ and inline elements contain _phrasing content_

> _flow_ content elements contain text or embedded content

and 

> _phrasing_ content elements define the text and the mark-up it contains.

It is a similar categorization, but there are now more categories of content.

**Comment by jessegreenberg** [comment]
Thanks for investigating @terracoda, we will remove the p tags inside buttons. I think the best thing to do is to remove default accessible content out of MultilineText because I imagine we  will end up removing the p tags around text more often than adding them in PhET sims because of cases like this.

**Comment by jessegreenberg** [comment]
Done in the above commits @terracoda, p tag removed from the button label.  Closing.
